### ISSUE TITLE
ClassConstantUsageCorrectnessInspector false positive

### ISSUE THREAD

**Issue by b1rdex** [kalessil/phpinspectionsea#411]
```php
abstract class BaseActiveRecord extends CActiveRecord
{
    public static function model($className = '')
    {
        if ($className === '' || $className === self::class || $className === parent::class) {
            $className = static::class;
        }

        /* @noinspection PhpIncompatibleReturnTypeInspection */
        return parent::model($className);
    }
```

`parent::class` has `parent` marked as error.

**Comment by kalessil** [comment]
Ooops, will fix this.
